.github/docker-compose.test.yml

```diff
@@ -14,15 +14,27 @@
 services:
   # The Unleash server contains the Unleash configuration and
   # communicates with server-side SDKs and the Unleash Proxy
+  frontend:
+    image: node:alpine
+    working_dir: /app
+    command: npx http-server ./build -p 80 --cors --proxy http://unleash:4242
+    ports:
+      - "3000:80"
+    volumes:
+      - ../frontend/build:/app:ro
```

Comment:
Frontend is built before running docker compose up (line 25 of .github/workflows/e2e.frontend.yaml)

Then it's served as static content and everything else is proxy-ed to unleash server

Reply:
If we have all node packages available, maybe we can run `start:prod --port 80` or `./node_modules/.bin/vite preview --port 80`?

Reply:
I considered it, but I think it is more realistic to use static files cause that's how we usually serve our frontend in production.